website/docs/docs/dbt-versions/experimental-features.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Preview new and experimental features in dbt Cloud"
+id: "experimental-features"
+sidebar_label: "Preview new dbt Cloud features"
+description: "Gain early access to many new dbt Labs experimental features by enabling this in your profile."
+---
+
+dbt Labs often beta tests features before deciding to continue on the [Product lifecycle](https://docs.getdbt.com/docs/dbt-versions/product-lifecycles#dbt-cloud).
+
+You can access experimental features if you want to preview beta features that haven’t yet been released to dbt Cloud. You can toggle on or off all experimental beta features in one place. As with other beta features, these features:
+
+- May not be feature-complete or fully stable as we’re actively developing them.
+- Could be discontinued at any time.
```

Comment:
Will they receive messaging about this in dbt Cloud if they have projects using these features?

Reply:
It sounds like eventually this will be the case but not with this initial launch. Here's info from @GideonShils 
> The plan is to design some new graphic (beta tag, etc) that will show a feature is not fully polished. Not sure that all features will have this necessarily though. Eventually we’ll list our features under the toggle too